kuka_omnimove_e575_support/urdf/caterpillar.xacro

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro" name="omnimove">
+  <xacro:macro name="omnimove" params="prefix">
+    prefix="${prefix}"
+      <link name="${prefix}base">
+        <visual>
+          <origin xyz="0 0 0" rpy="0 0 0" />
+          <geometry>
+            <mesh filename="package://kuka_omnimove_e575_support/meshes/caterpillar_simplified.dae" />
```

Comment:
in all other packages, we have separate folders for visual and collision meshes, to make things more straighforward. It would be nice if you could also create these 2 folders here